ironfish-rust/src/mining/mine.rs

```diff
@@ -77,6 +81,50 @@ mod test {
         assert_eq!(result.unwrap(), 43);
     }
 
+    #[test]
+    fn test_mine_batch_step_size() {
```

Comment:
This test seems to pass on the old code as well -- I might be misunderstanding which issue it's fixing, but could you modify the test so it fails on the old code?